lib/adapters/QUnitAdapter.js

```diff
@@ -178,12 +179,23 @@ export default class QUnitAdapter extends EventEmitter {
 
   onTestStart (details) {
     this.errors = []
+    this.assertions = []
   }
 
   onLog (details) {
-    if (!details.result) {
-      this.errors.push(details)
+    var assertion = {
+      success: details.result,
+      actual: details.actual || null,
```

Comment:
This is wrong. Anything falsy is valid here and should not be replaced with `null`. I'd remove the fallback, to keep whatever the property was.